test_integration/mosaic-create/Utils.ts

```diff
@@ -23,7 +23,7 @@ export default class Utils {
         '8546:8546',
         '--name',
         'integration_test_origin',
-        'mosaicdao/dev-chains',
+        'mosaicdao/dev-chains:1.0.0',
```

Comment:
Do we always need to specify docker version?
Shouldn't dev chain use latest published docker. Facilitator can access specific npm version of mosaic chains.

Reply:
Yes, Because given version of mosaic-chains will be compatible with specific version of dev-chain.